protostar/cli/signer_factory.py

```diff
@@ -0,0 +1,100 @@
+import importlib
+import os
+from pathlib import Path
+from typing import Optional
+
+from starknet_py.net.models import StarknetChainId
+
+from protostar.cli import Command
+from protostar.protostar_exception import ProtostarException
+from protostar.starknet_gateway import (
+    AccountConfig,
+    ProtostarBaseSigner,
+    create_protostar_default_signer,
+)
+
+PRIVATE_KEY_ENV_VAR_NAME = "PROTOSTAR_ACCOUNT_PRIVATE_KEY"
+
+
+SIGNABLE_ARGUMENTS = [
+    Command.Argument(
+        name="account-address",
+        description="Account address",
+        type="str",
+    ),
+    Command.Argument(
+        name="private-key-path",
+        description="Path to the file, which stores your private key (in hex representation) for the account. \n"
+        f"Can be used instead of {PRIVATE_KEY_ENV_VAR_NAME} env variable.",
+        type="path",
+    ),
+    Command.Argument(
+        name="signer-class",
+        description="Custom signer class module path.",
+        type="str",
+    ),
+]
+
+
+def create_account_config_from_args(
```

Comment:
Hmm, I should have picked a better title for this PR. It's more about extracting `AccountConfig` model. I left the `args` here because this function uses `create_custom_signer` and `get_private_key` which is IO.